content/v2.0/cloud/get-started.md

```diff
@@ -18,29 +18,34 @@ _See the differences between {{< cloud-name "short">}} and InfluxDB OSS
 [below](#differences-between-influxdb-cloud-and-influxdb-oss)._
 
 ## Start for free
+
 Start using {{< cloud-name >}} at no cost with the [Free Plan](/v2.0/cloud/pricing-plans/#free-plan).
 Use it as much and as long as you like within the plan's rate-limits.
 Limits are designed to let you monitor 5-10 sensors, stacks or servers comfortably.
 Once you're ready to grow, [upgrade to the Usage-based Plan](/v2.0/cloud/account-management/upgrade-to-usage-based-plan/).
 
 ## Sign up
 
-1. Go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your email address and password, and click **Sign Up**.
-2. InfluxDB Cloud requires email verification to complete the sign up process. Verify your email address by opening
-   the email sent to the address you provided and clicking **Verify Your Email**.
-3. Choose your cloud provider. Currently, **AWS** is the only cloud provider available.
+1. Choose one of the following:
+   - To subscribe for a plan through InfluxData, go to [InfluxDB Cloud 2.0]({{< cloud-link >}}), enter your credentials, and then click **Sign Up**.
+   - To subscribe for a plan through AWS Marketplace, go to [Amazon Marketplace](https://aws.amazon.com/marketplace), search for **InfluxDB Cloud 2.0**, and then follow the prompts to subscribe. After you click **Set Up Your Account**, enter your credentials, and then click **Sign Up**.
+
+       {{%note%}} **Note:** You can sign up for a new InfluxDB Cloud 2.0 account through **either** InfluxData or AWS Marketplace. Currently, we do **not support** using an existing InfluxDB Cloud 2.0 account to sign up for an AWS Marketplace subscription.
+        {{%/note%}}
+
+2. InfluxDB Cloud requires email verification to complete the sign up process.
+   Verify your email address by opening the email sent to the address you provided and clicking **Verify Your Email**.
+3. (InfluxData subscriptions only) Choose your cloud provider. Currently, **AWS** is the only cloud provider available.
```

Comment:
I wonder if we even need to call out that AWS is the only provider available here? There are others coming VERY soon.

Reply:
I think it'd be fine to leave it out given GCP is coming so soon.

Reply:
sounds good! removed